pajbot/managers/irc.py

```diff
@@ -129,8 +129,8 @@ def _can_send(self, is_whisper):
         if is_whisper:
             return (
                 self.num_privmsg_sent < self.bot.tmi_rate_limits.privmsg_per_30
-                and self.num_whispers_sent_second < self.bot.tmi_rate_limits.whispers_per_second
-                and self.num_whispers_sent_minute < self.bot.tmi_rate_limits.whispers_per_minute
+                and self.num_whispers_sent_second < 2
+                and self.num_whispers_sent_minute < 90
```

Comment:
These values should still be stored as a variable. They make sense to be in the `tmi_rate_limits` object.

Reply:
What is the point of storing them as variables if the rate limit is set for all accounts? Doesn't it seem a bit redundant? Since the `num_whispers_sent_second` and `num_whispers_sent_minute` variables are not referenced anywhere else?

Reply:
Don't introduce new "magic numbers" into the code that will be harder to find and modify in the future

https://www.perforce.com/resources/qac/high-integrity-cpp-coding-standard-expressions